src/DotNetCore.CAP.AzureServiceBus/CAP.AzureServiceBusOptions.cs

```diff
@@ -25,6 +24,11 @@ public class AzureServiceBusOptions
         /// </summary>
         public string ConnectionString { get; set; } = default!;
 
+        /// <summary>
+        /// Namespace of Servicebus
+        /// </summary>
+        public string Namespace { get; set; } = default!;
```

Comment:
Hi! For these new properties you introduced here, it would be good to add some comment explaining their purpose and also [update the documentation with it](https://github.com/dotnetcore/CAP/blob/master/docs/content/user-guide/en/transport/azure-service-bus.md).